### DISCUSSION TITLE
helloSystem 0.6.0 (12.2-RELEASE r366954): latest packages

### DISCUSSION THREAD

**Discussion by grahamperrin** [helloSystem/hello#222] [General]
https://github.com/helloSystem/hello/discussions/219#discussioncomment-1489425

> 𠉥… I don't remember that we used latest packages for anything but very experimental FreeBSD 13 based builds.

@probonopd please re-check the outcome of an installation from `hello-0.6.0_0F54-FreeBSD-12.2-amd64.iso`.

Here, after restoring a VirtualBox snapshot that I performed after installation, I have _latest_: 

![image](https://user-images.githubusercontent.com/192271/137636545-0b182705-a909-40e7-b552-939c66286567.png)

    uname -aKU
    FreeBSD mowa219-gjp4-hello-060-vm 12.2-RELEASE FreeBSD 12.2-RELEASE r366954 GENERIC  amd64 1202000 1202000

**Comment by probonopd** [comment]
Yes, that was in https://github.com/helloSystem/ISO/releases/tag/r0.6.0, see "Known Issues".
0.7.0 experimental builds no longer use `latest`.

**Reply by grahamperrin** [reply]
Thanks. So, working as intended: 

* packages from _latest_ are the norm, expected, for release 0.6.0

– for users of live systems and installed systems. 

https://pkg-status.freebsd.org/builds?type=package&jailname=122amd64 ▶ default ports ▶ http://beefy6.nyi.freebsd.org/jail.html?mastername=122amd64-default ▶ all builds ▶ **the highlighted build** ▶

* at the time of writing, _stopped:done:_ http://beefy6.nyi.freebsd.org/build.html?mastername=122amd64-default&build=dd3b4f6d0e27
